components/NewOutivityForm/index.js

```diff
@@ -0,0 +1,155 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function NewOutivityForm({
+  CreateOutivity,
+  handleCancel,
+  selectedImage,
+}) {
+  return (
+    <main>
+      <StyledNewOutivitiesForm onSubmit={CreateOutivity}>
+        <h1>New Outivity</h1>
+        <StyledNewOutivitiesFormFields>
+          <StyledNewOutivitiesFormField>
+            <label htmlFor="image">Image</label>
+            {selectedImage && (
+              <Image
+                src="/{selectedImage}"
+                alt="outivity-image"
+                width={300}
+                height={200}
+              />
+            )}
+            <StyledNewOutivitiesFormInput
+              type="text"
+              name="outivityImage"
+              id="outivityImage"
+              placeholder="Insert a url from unsplash..."
+              required
+            />
+          </StyledNewOutivitiesFormField>
+          <StyledNewOutivitiesFormField>
+            <label htmlFor="outivitiy">Outivity</label>
+            <StyledNewOutivitiesFormInput
+              type="text"
+              name="outivityName"
+              id="outivityName"
+              placeholder="Type in a name..."
+              required
+            />
+          </StyledNewOutivitiesFormField>
+          <StyledNewOutivitiesFormField>
+            <label htmlFor="area">Area</label>
+            <StyledNewOutivitiesFormInput
+              type="text"
+              name="outivityArea"
+              id="outivityArea"
+              placeholder="In which city/area is it located?"
+              required
+            />
+          </StyledNewOutivitiesFormField>
+          <StyledNewOutivitiesFormField>
+            <label htmlFor="country">Country</label>
+            <StyledNewOutivitiesFormInput
+              type="text"
+              name="outivityCountry"
+              id="outivityCountry"
+              placeholder="In which country?"
+              required
+            />
+          </StyledNewOutivitiesFormField>
+          <StyledNewOutivitiesFormField>
+            <label htmlFor="description">Description</label>
+            <StyledNewOutivitiesFormTextarea
+              type="text"
+              name="outivityDescription"
+              id="outivityDescription"
+              placeholder="Type in some info..."
+              rows="6"
+              required
+            />
+          </StyledNewOutivitiesFormField>
+          <StyledNewOutivitiesFormSpan>
+            <StyledCancelButton onClick={handleCancel}>
```

Comment:
this button needs a type="button", otherwise it will be treated as a submit button, since it is part of a form.

Without the type, the current behaviour is not ideal for the user, if they want to cancel the form submission.
After clicking on "cancel", they will stay on the page, but your page tries to submit the form and this will result in your input fields being checked. The user will either get the note to fill out a field, or the data will be submitted, depending on whether all fields were filled out or not.

```suggestion
            <StyledCancelButton onClick={handleCancel} type="button"> 
```